cms/src/components/content/ContentRow.vue

```diff
@@ -22,10 +22,10 @@ type Props = {
     parent: PostDto | TagDto;
     parentType: DocType.Post | DocType.Tag;
     language: Uuid;
```

Comment:
Rename ```language``` prop to languageId to avoid confusion. I know this was not your code, but will help for clarity.